namara/__init__.py

```diff
@@ -10,7 +15,34 @@ def __init__(self, api_key, debug=False, host='https://api.namara.io', api_versi
         self.api_version = api_version
         self.base_path = '{0}/{1}'.format(self.host, self.api_version)
         self.headers = {'Content-Type': 'application/json', 'X-API-Key': api_key}
-
+    
+    def export(self, dataset_id, organization_id, options=None, output_format='url', output_file=None):
+        url = self.get_url(f'/data_sets/{dataset_id}/data/export?geometry_format=wkt&organization_id={organization_id}')
+        while True: 
+            response = self.__session.get(url, params=options, headers=self.headers).result().json()
+            if self.debug: 
+                logging.debug('Response Object: ' + response)
+            if 'message' in response and response['message'] == 'Exported':
+                if output_format == 'url': 
+                    return response['url']
+                elif output_format == 'dataframe': 
+                    list_of_chunks = [] 
+                    for chunk in pd.read_csv(response['url'], chunksize=100):
+                        list_of_chunks.append(chunk)
+                    return pd.concat(list_of_chunks)
+                elif output_format == 'csv': 
+                    with requests.get(response['url'], stream=True) as r:
+                        shutil.copyfileobj(r.raw, output_file)
+                    return
+                else: 
+                    raise Exception('`output_format` param must be "csv", "dataframe", or "url" (default)')
+            elif 'message' in response and response['message'] == 'Failed':
+                raise Exception(f"Could not export dataset {dataset_id}")
+            elif 'message' in response and response['message'] == 'Pending': 
```

Comment:
I think "Exporting" is a potential state as well.

Reply:
Good catch